docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -166,6 +166,14 @@ Dates listed indicate the start date of the maintenance cycle.
      - End Of Life
      - Control Node Python
      - Target Python / PowerShell
+   * - `2.19`
+     - GA: 16 June 2025
+       Critical: 03 Nov 2025
+       Security: 18 May 2026
```

Comment:
```suggestion
     - | GA: 16 June 2025
       | Critical: 03 Nov 2025
       | Security: 18 May 2026
```

This is currently mis-formatted:

![image](https://github.com/user-attachments/assets/e4c6b2bc-f44a-411f-907f-9bd9f9cffcb0)